### ISSUE TITLE
WebGPU support

### ISSUE THREAD

**Issue by redstrate** [redstrate/prism#10]
I've been thinking of adding WebGPU support because being able to run Prism on the web, even a simple version just for demos would be awesome. Currently on hold for now because of lack of stability and waiting on getting the GFX api redone.

(This issue is just tracking *WebGPU* support, not Web support which hasn't even started yet.)

**Comment by redstrate** [comment]
Added a placeholder for WebGPU in c9cac0f!

**Comment by redstrate** [comment]
Work has started on the `master` branch!
